core/src/main/scala/kafka/server/RequestHandlerHelper.scala

```diff
@@ -107,6 +107,18 @@ class RequestHandlerHelper(
 
   // Throttle the channel if the request quota is enabled but has been violated. Regardless of throttling, send the
   // response immediately.
+  def sendResponse(
```

Comment:
I could not call it `sendResponseMaybeThrottle` because the Scala compiler is confused and cannot disambiguate it from the existing `sendResponseMaybeThrottle`.

Reply:
How about `sendMaybeThrottle` or `sendAndMaybeThrottle`? I think including "Response" is redundant given the parameters.

Reply:
`sendMaybeThrottle` sounds good to me. Thanks for the suggestion!